test/local/push_tests.js

```diff
@@ -55,7 +55,7 @@ var mockDbResult = {
 test(
   'pushToDevices throws on device not found',
   function (t) {
-    var push = require('../../lib/push')(mockLog(), mockDbEmpty)
+    var push = require('../../lib/push')(mockLog(), mockDbEmpty, {})
```

Comment:
nit: maybe we can just do `mockConfig = {}` up top in the file for our future config updates